mimecast/tests/get_managed_url.json

```diff
@@ -2,16 +2,21 @@
     "body": {
         "action": "get_managed_url",
         "connection": {
-            "app_id": null,
+            "app_id": "83ef489c-fdbf-4cca-afd5-5d2bb5350ace",
             "app_key": {
-                "secretKey": null
+                "secretKey": "44e13412-7049-4826-8736-05fc1ccff432"
             },
-            "auth_type": null,
             "credentials": {
-                "password": null,
-                "username": null
+                "password": "L6+Q4g[{P=BmZBM",
+                "username": "test@rapid7.com"
             },
-            "url": null
+            "url": "https://sandbox-api.mimecast.com",
+	    "access_key" : {
+                "secretKey": "mYtOL3XZCOwG96BOiFTZRiC5rdvDmP4FFdwU2Y1DC1Us-nk2JcI7ynUrZ9aEuzQMV7pPWWxTnPVtsJ6x3fajAh3cRskP0w8hNjaFFVkZB6EFLSKjzsvEO47qNibOVexA"
```

Comment:
@tylerslijboom-r7 These creds are now compromised by being exposed in this open-source repo. Password reset forthcoming.